values.yaml

```diff
@@ -82,15 +82,15 @@ mastodon:
     # -- (Sidekiq Container) Security Context for all Pods, overwrites .Values.securityContext
     securityContext: {}
     # -- Resources for all Sidekiq Deployments unless overwritten
```

Comment:
if we set default values we could probably drop `.Values.resources`

Reply:
Done!